certbot/tests/display/ops_test.py

```diff
@@ -474,7 +474,7 @@ def test_choose_names_success(self, mock_util):
         result = self._call(items, None)
         self.assertEqual(result, [items[2]])
         self.assertIs(mock_util().checklist.called, True)
-        self.assertIsNone(mock_util().checklist.call_args[0][0])
+        self.assertIs(mock_util().checklist.call_args[0][0], "")
```

Comment:
Shouldn't this be `assertEqual`? `assertIs` is just for `True/False`.

Reply:
Thanks, this is fixed.